quickstart.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+# Exposing ports
+EXPOSE 6900
+
+# Environment variables
+ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/users.yml
+ENV UVICORN_PORT=6900
+
+# Install Python
+RUN apt update
+RUN apt -y install curl python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+RUN python3.9 get-pip.py
+
+# Install argilla
+RUN pip install argilla[server]
```

Comment:
This will install the latest published release. We should have something similar to the `release.Dockerfile` image. Copy the generated wheels packages and install them locally.

Reply:
Sure I'll use that